### PR TITLE
Performant import of production data

### PR DESCRIPTION
**Pull request by roschaefer**
@appinteractive thanks for pointing out `split`. You just saved me some
days of work to refactor the import statements to use CSV instead of
JSON files.

@Tirokk when I enter `:schema` in Neo4J web UI, I see the following:
```
:schema
Indexes
   ON :Badge(id) ONLINE
   ON :Category(id) ONLINE
   ON :Comment(id) ONLINE
   ON :Post(id) ONLINE
   ON :Tag(id) ONLINE
   ON :User(id) ONLINE

No constraints
```

So I temporarily removed the unique constraints on `slug` and added
plain indices on `id` for all relevant node types. We cannot omit the
`:Label` unfortunately, neo4j does not allow this. So I had to add all
indices for all known node labels instead.

With indices the import finishes in:
```
Time elapsed: 351 seconds
```
:tada:

@appinteractive when I keep the unique indices on slug, I get an error
during import that a node with label `:User` and slug `tobias` already
exists. Ie. we have unqiue constraint violations in our production data.

@mattwr18 @ulfgebhardt @ogerly I started the application on my machine
on the production data and it turns out that the index page
http://localhost:3000/ takes way to long. Visiting my profile page at
http://localhost:3000/profile/5b1693daf850c11207fa6109/robert-schafer
is fine, though. Even pagination works. When I visit a post page with
not too many comments, the application is fast enough, too:
http://localhost:3000/post/5bbf49ebc428ea001c7ca89c/neues-video-format-human-connection-tech-news

### Issues

- [X] None

### Checklist

- [X] None

### How2Test

- [X] None

### Todo

- [X] None

### CODE REVIEW

**Comment by roschaefer** [comment]
Did some query profiling on the generated cypher statement of [this graphQL query](https://github.com/Human-Connection/Human-Connection/blob/master/webapp/pages/index.vue#L94):


plan

**Comment by roschaefer** [comment]
Here are our duplicates in the production data:

## Users (34 duplicates)
```cypher
match (u:User)
match (u2:User)
where u.slug = u2.slug and u <> u2
return u.slug, u2.slug
```

|           |           | 
|-----------|-----------| 
| u.slug    | u2.slug   | 
| andre     | andre     | 
| pat       | pat       | 
| jurgen    | jurgen    | 
| manuel    | manuel    | 
| christoph | christoph | 
| maria     | maria     | 
| marion    | marion    | 
| sebastian | sebastian | 
| tobias    | tobias    | 
| susanne   | susanne   | 
| marvin    | marvin    | 
| wolf      | wolf      | 
| lars      | lars      | 
| andy      | andy      | 
| sonja     | sonja     | 
| guido     | guido     | 
| max       | max       | 
| tobias    | tobias    | 
| sebastian | sebastian | 
| andy      | andy      | 
| marvin    | marvin    | 
| manuel    | manuel    | 
| jurgen    | jurgen    | 
| max       | max       | 
| lars      | lars      | 
| sonja     | sonja     | 
| andre     | andre     | 
| maria     | maria     | 
| christoph | christoph | 
| wolf      | wolf      | 
| guido     | guido     | 
| susanne   | susanne   | 
| pat       | pat       | 
| marion    | marion    | 


The positive message: There seem to be no duplicates for label types `Comment`, `Post` or `Category`.

@appinteractive I had a quick look into the returned records. The duplicates have distinct email addresses, so I believe the user accounts are not used by the same person. The users are neither `disabled` nor `deleted`. 

Do you have an idea why we have duplicate users in the database?

Do you think it is safe to "repair" those unique constraint violations manually?

**Comment by appinteractive** [comment]
@roschaefer glad that I could help with the import! 

The performance issue on the newsfeed comes from the sorting by date! At least that’s the issue I encountered but did not had a solution. Maybe we need to use another date format there or some index!?

The dublicated slugs are strange, maybe they are related to invites!? Maybe appand a number to the slug of the least active users?

File: deployment/legacy-migration/maintenance-worker/migration/mongo/import.sh [hunk 1]
```diff
@@ -9,16 +9,17 @@ echo "MONGODB_DATABASE         ${MONGODB_DATABASE}"
...
+mkdir -p /tmp/mongo-export/
 
 ssh -4 -M -S my-ctrl-socket -fnNT -L 27018:localhost:27017 -l ${SSH_USERNAME} ${SSH_HOST}
 
 for collection in "categories" "badges" "users" "contributions" "comments" "follows" "shouts"
 do
-  mongoexport --host localhost -d ${MONGODB_DATABASE} --port 27018 --username ${MONGODB_USERNAME} --password ${MONGODB_PASSWORD} --authenticationDatabase ${MONGODB_AUTH_DB} --db ${MONGODB_DATABASE} --collection $collection --out "/tmp/mongo-export/$collection.json"
+  mongoexport --db ${MONGODB_DATABASE} --host localhost -d ${MONGODB_DATABASE} --port 27018 --username ${MONGODB_USERNAME} --password ${MONGODB_PASSWORD} --authenticationDatabase ${MONGODB_AUTH_DB} --collection $collection  --collection $collection --out "/tmp/mongo-export/$collection.json"
+  mkdir -p /tmp/mongo-export/splits/$collection/
+  split -l 500 /tmp/mongo-export/$collection.json /tmp/mongo-export/splits/$collection/
```
Comment (appinteractive) [inline deployment/legacy-migration/maintenance-worker/migration/mongo/import.sh]: Slit is limited to max 676 parts as it used two letters, you should append -a 3 which used three letters. Also go with maybe 1000 lines?

Comment (roschaefer) [reply deployment/legacy-migration/maintenance-worker/migration/mongo/import.sh]: Done!

**Pull request merged**